Registered k8s.io/deprecated & k8s.io/removed-release audit annotation (#47268)
kubernetes/website: content/en/docs/reference/labels-annotations-taints/audit-annotations.md

<!-- body -->

## k8s.io/deprecated

Example: `k8s.io/deprecated: "true"`

> Value **must** be boolean. It is set to "true" indicating the request used a deprecated API version.

Reviewer: ```suggestion
Value **must** be "true" or "false". The value "true" indicates that the request used a deprecated API version.
```

Author: That looks good. I'll make the changed later today. Just waiting in case more reviews are made.

If a target removal release is identified, the audit event includes the `k8s.io/removed-release` annotation.

## k8s.io/removed-release

> Example: `k8s.io/removed-release: 1.22`

Reviewer 2: ```suggestion
Example: `k8s.io/removed-release: "1.22"`
```
?

Author: Right. i missed this.
